pcsx2/gui/MainFrame.cpp

```diff
@@ -450,7 +450,8 @@ void MainEmuFrame::CreateWindowsMenu()
 void MainEmuFrame::CreateCaptureMenu()
 {
 	m_menuCapture.Append(MenuId_Capture_Video, _("Video"), &m_submenuVideoCapture);
-	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Screenrecorder"));
+	m_submenuVideoCapture.Append(MenuId_Capture_Video_Record, _("Start Screenrecorder"),	
+		_("Creates a video-only file and an audio-only wav file."));
```

Comment:
Not really needed.

Reply:
Well the tooltip could lose partly it's meaning when GSdx/SPU2-x get merged.